### ISSUE TITLE
App fails to find resources when running in Firefox, Safari, or Edge

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/create-app#97]
The same problem occurs in all of the three browsers
![Screen Shot 2021-03-15 at 2.14.01 PM.jpg](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/aa95ace0-866c-41c4-943c-c181bccd67c6)

**Comment by PhotoNomad0** [comment]
@cckozie - I think the only org that works now is `test_org` - a TSV thing.  Anyway with test_org this works for me in firefox and safari.

**Comment by PhotoNomad0** [comment]
@cckozie it looks like https://git.door43.org/translate_test does not have en_ult or en_ust repos in it.

**Comment by mannycolon** [comment]
> @cckozie - I think the only org that works now is `test_org` (a TSV thing). I don't have access to `translate_test` (in create_app). Anyway with test_org this works for me in firefox and safari.

Same here, both browsers work for me with `test_org`

**Comment by cckozie** [comment]
@PhotoNomad0 @mannycolon - This is very bizarre! In all of the testing I've done (99% on Chrome) I've always used the test_org and have never gotten caught trying to use translate_test. Yet when I now log into create 2.0 on Firefox and Safari and go to the account settings, in both cases I see translate_test as the org. When I went to my Windows machine where the Edge session was still open, when I go to account settings the org field is empty. When I open the org dropdown I see the two orgs and translate-test is the top one which I suspect is the one it was using. Even though I have a really old brain, it's hard for me to believe I made the same mistake three different times (separated by hours), and only those 3 times. However, I don't see any other explanation unless there is some way with a never before used browser for the app that when logging in it somehow bypassed the setting screen and defaulted it to the last used ones from tC Create 1.0. For now I'm chalking it up to user error.
